lec14-git.Rmd

```diff
@@ -1,4 +1,146 @@
 ---
 title: "Scientific collaboration, Git, and GitHub"
+author: "Elliott Sales de Andrade"
 ---
 
+## Lesson preamble
+
+> ### Lesson objectives:
+>
+> - Practice using the 4 main Git commands and the Git-RStudio interface
+> - Learn how and when to use branches
+> - Learn how to effectively use Pull Requests
+> - Project work
+>
+> ### Lesson outline:
+>
+> - Review basic Git usage (15 min)
+> - Branching in Git (30 min)
+> - Pull requests on GitHub (30 min)
+> - Project work
+>     - Continue working on assignment 7 (20 min)
+
+-----
+
+## Git
+
+### Recap
+
+#### Shell commands
+
+What do these do?
+
+- `ls`
+- `cd`
+- `touch`
+- `echo`
+- `>`
+- `cat`
+
+#### Setup
```

Comment:
Are you getting them to do this again? Oooh, right for the people on the lab computers... That's annoying...